README.md

```diff
@@ -31,3 +31,41 @@ This app also supports Hot Module Reloading for both the React Components and Re
 `./scripts/client-dev.sh`
 
 Note: You may see an error regarding Routes on each change, this will be the case until React router v4 is released.
+
+## The Publishing Process for Atoms
+This is the publishing process for Atom Workshop and recommended best practices for other Tools that create Atoms
+
+This project uses the [atom maker](https://github.com/guardian/atom-maker/) for publishing, reindexing and saving atoms in a
+DynamoDB. As a result there is the concept of Drafts (atoms where all fields are optional) and Atoms where whether or not a
+field is optional depends on the [thrift definition](https://github.com/guardian/content-atom).
+### Publishing flow
+1. Create atom
+2. The atom is saved in Draft form in the DraftsDynamoDataStore while the user has not filled out all the fields required according to the thrift
+definition. This enables the user to save their work at any point. At this point in the flow the atom is not in CAPI.
+3. Every time an atom is saved to the DraftsDynamoDataStore atom workshop also checks if all the required fields have been
+supplied. If they have the atom is put in PreviewDynamoDataStore and put into preview CAPI. This step is not visible to the user.
+4. The atom is now in a publishable state the 'publish' button in the UI becomes active.
+5. On selecting publish the atom is put in the PublishedDynamoDataStore and in live CAPI. Once published the atom can be embedded in
+composer content.
+
+### Editing Rules
```

Comment:
I agree, I think having what the differences between draft, preview and live is useful. In reading the below steps I had to remind myself why we have preview and it's usecase.